axelrod/strategies/oncebitten.py

```diff
@@ -111,3 +111,14 @@ def strategy(self, opponent):
     def reset(self):
         self.D_count = 0
         self.history = []
+
+
+class BackStabber(FoolMeOnce):
+
+    name = 'BackStabber'
+
+    def strategy(self, opponent):
+        if len(opponent.history) > 196:
```

Comment:
You say in the documentation:

> A variation of Fool Me Once, this strategy will also defect on the final 2 turns of the round unconditionally.

This doesn't quite line up with what is here (final 4 turns). Also could you change 'round' to 'duel'.

The other issue is that using `196` assumes the tournament length is 200 which it does not necessarily have to be. It would be nicer if you coded something generic that always did 4 rounds less than the length of the tournament itself (although that might need some tweaking as I'm not sure the players have access to the tournament length).

(An easier/hackier fix would be to say in the description that the strategy defects after 196 rounds.)